mesh/node.c

```diff
@@ -1686,6 +1735,50 @@ void node_join(const char *app_path, const char *sender, const uint8_t *uuid,
 					req, l_free);
 }
 
+static bool node_import_full(const char *app_path, const char *sender,
+					void *parsed_node, const uint8_t *uuid,
+					node_ready_func_t cb, void *user_data)
+{
+	l_debug("");
+	return false;
+}
+
+static bool node_import_simplified(const char *app_path, const char *sender,
+					void *parsed_node, const uint8_t *uuid,
+					node_ready_func_t cb, void *user_data)
+{
+	struct managed_obj_request *req = l_new(struct managed_obj_request, 1);
+	struct node_import_request *node = l_new(struct node_import_request, 1);
+
+	l_debug("");
+	node->user_data = user_data;
+	node->i_node = parsed_node;
+	memcpy(node->uuid, uuid, 16);
+
+	req->data = (void *) uuid;
+	req->user_data = node;
+	req->cb = cb;
+	req->type = REQUEST_TYPE_IMPORT;
+
+	l_dbus_method_call(dbus_get_bus(), sender, app_path,
+						L_DBUS_INTERFACE_OBJECT_MANAGER,
+						"GetManagedObjects", NULL,
+						get_managed_objects_cb,
+						req, l_free);
+	return true;
+}
+
+bool node_import(const char *app_path, const char *sender, void *parsed_node,
+		const uint8_t *uuid, node_ready_func_t cb, void *user_data)
+{
```

Comment:
Rename `parsed_node` to `import` and cast to real type before passing to `_simplified`/`_full`